src/event.cpp

```diff
@@ -116,7 +116,9 @@ void event::actualize()
         if (g->u.has_amount("petrified_eye", 1)) {
             add_msg(_("The eye you're carrying lets out a tortured scream!"));
             sounds::sound(g->u.pos(), 60, "");
-            g->u.add_morale(MORALE_SCREAM, -15, 0, 300, 5);
+            if (g->u.is_deaf() == false) {
```

Comment:
hmm... but i don't want anything to happen if that check is true.. so i'd have to also add an 'else' clause. Would that be desirable?